### PR TITLE
chore: Update components to 18, and bump version

### PR DESCRIPTION
**Pull request by getnashty**
# Trello card link

# Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published

## Acceptance

https://www.notion.so/aiera/Test-Flows-Template-5203736ccfcb424c81643065f2bbf5e2

- [x] I have reviewed the Test Flows template and it is up-to-date after considering my changes

### CODE REVIEW

**Review by mmichael0413 — APPROVED** [review]
💥

File: package.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
         "pre-commit": "npm run build:lib:types && concurrently lint-staged npm:test"
     },
     "peerDependencies": {
-        "react": "17.0.2",
-        "react-dom": "17.0.2"
+        "react": "^18.3.1",
```
Comment (mmichael0413) [inline package.json:42]: We should actually lock these down - please remove the `^` for these two

File: package.json [hunk 2]
```diff
@@ -92,14 +91,14 @@
         "postcss": "^8.3.6",
         "postcss-load-config": "^3.1.0",
         "prettier": "^2.3.2",
-        "react": "17.0.2",
-        "react-dom": "17.0.2",
+        "react": "^18.3.1",
```
Comment (mmichael0413) [inline package.json:94]: Same comment as above re: locking `react` and `react-dom` versions

File: package.json (hunk 1 above)
Comment (bcantello) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (bcantello) [reply package.json]: done

**Comment by bcantello** [comment]
@mmichael0413 @getnashty This is ready for another review. Live updates working, all tests passing, gpg keys...keying

**Comment by getnashty** [comment]
I am not able to approve because I created the PR, but I do in fact approve 👍

**Review by bcantello — APPROVED** [review]

**Pull request merged**